client/index.html

```diff
@@ -2,10 +2,16 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <link rel="stylesheet" href="./index.css">
-  <script type="module" src="./index.js" defer></script>
   <meta name="color-scheme" content="dark light">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="A page to display compatible browsers from a browserslist string">
+  <!-- TODO Add og.png (1200x630) <meta property="og:image" content="/og.png"> -->
+  <link rel="icon" href="/favicon.ico" sizes="any">
+  <link rel="icon" href="/assets/icon.svg" type="image/svg+xml">
```

Comment:
Why SVG without cache buster? We can use `./base/icon.svg` as we have on OKLCH Color Picker

Reply:
The same for `apple-touch-icon`.

Reply:
did it for oklch-picker too

and also noticed a bug there :)

https://github.com/evilmartians/oklch-picker/pull/56